.circleci/config.yml

```diff
@@ -1510,7 +1510,7 @@ commands:
             fi
             if [[ "${CIRCLE_BRANCH}" == "master" || -n "${CIRCLE_TAG}" ]]; then
               echo "On master, running all QA tests..."
-              make -C qa-tests-backend test || touch FAIL
+              make -C qa-tests-backend test FAIL_FAST=true || touch FAIL
```

Comment:
I think we shouldn't do this for RC and release builds.

Reply:
Fixed